src/common/ipc/cluster/index.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * This channel is broadcast on whenever the cluster fails to list namespaces
+ * during a refresh and no `accessibleNamespaces` have been set.
+ */
+export const ClusterListNamespaceForbiddenChannel = "cluster:list-namespace-forbidden";
+
+export type ListNamespaceFordiddenArgs = [clusterId: string];
```

Comment:
I recently learned that you can do this for tuple types. The actual type is still just `[string]` but it is a good documentation in code.